draft-ietf-intarea-provisioning-domains.md

```diff
@@ -332,11 +333,13 @@ In other words, different Explicit PvDs MAY be advertised with RAs using
 the same link-local source address; but different Implicit PvDs, advertised
 by different RAs, MUST use different link-local addresses because
 these Implicit PvDs are identified by the source addresses of the
-RAs.
+RAs. If a link-local address on the router is changed, then any new RA
+sent from the changed link-local address without an Explicit PvD option
+will be interpreted as a new Implicit PvD by PvD-aware hosts.
 
-As specified in {{!RFC4861}}, when the set of options
+As specified in {{!RFC4861}} and {{!RFC6980}}, when the set of options
 causes the size of an advertisement to exceed the link MTU, multiple
-router advertisements can be sent, each containing a subset of the
+router advertisements MUST be sent, each containing a subset of the
```

Comment:
s/router advertisements MUST be sent/router advertisements MUST be sent to avoid fragmentation/